src/actions/breakpoints/addBreakpoint.js

```diff
@@ -28,18 +30,21 @@ async function addBreakpointPromise(getState, client, sourceMaps, breakpoint) {
   };
```

Comment:
For instance, if you run Flow, the first error in this file is the 2 lines above here, because `source` here could technically be `null` if the `breakpoint.location.sourceId` was actually for a source that it couldn't find.

If I were writing this fresh, I personally might just make `getSource` throw instead of returning `null`, but that's probably too aggressive of a change for you to make, so instead I'd probably add
```diff
  const source = getSource(state, breakpoint.location.sourceId);
+ if (!source) {
+   throw new Error(`Failed to add breakpoint because ${breakpoint.location.sourceId} is unknown`);
+ }
```
because this means that Flow can know for sure that `source` _must_ point at a valid source file by the time it has to read `source.id`.

Adding Flow to an existing codebase often comes down to 

1. Adding annotations where strictly needed
2. Tweaking your code to prove to Flow that the types actually fit together